src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -37,24 +38,14 @@ default boolean isSameStateAs(ReadOnlyPerson other) {
      */
     default String getAsTextShowAll() {
```

Comment:
does this really show all if the `getPrintableString` evaluates the `isPrivate` variable?